### PR TITLE
Completed javascript-if exercise - Justin Song

### PR DESCRIPTION
**Pull request by justinsong1608**

### CODE REVIEW

File: javascript-if/main.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  return false;
+}
+
+function categorizeAcidity(pH) {
+  if (pH === 7) {
+    return 'netural';
+  } else if (pH < 7 && pH > 0) {
```
Comment (shawnkost) [inline javascript-if/main.js]: This should be greater than or equal to. `pH >= 0`

File: javascript-if/main.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    return 'acid';
+  } else if (pH < 14 && pH > 7) {
```
Comment (shawnkost) [inline javascript-if/main.js]: This should be less than or equal to. `pH <= 14`

**Review by shawnkost — CHANGES_REQUESTED** [review]
A couple changes requested below Justin!

**Comment by justinsong1608** [comment]
Updated!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**